src/ProcessRunner.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace JeroenG\Packager;
+
+trait ProcessRunner
```

Comment:
Nice refactor 👍

Reply:
Glad you like it. I considered calling the other one `ComposerCommander` or something like that, but decided to keep it boring...